### ISSUE TITLE
Who uses prettier?

### ISSUE THREAD

**Issue by leipert** [prettier/prettier#1351]
First of all thanks for the latest stable releases 💘 

Would it make sense to add a *Who uses prettier?* section to the readme?

@vjeux mentions in [his talk](https://www.youtube.com/watch?v=0Q4kUNx85_4) that it might be hard to convince your team to use prettier. Maybe it is easier to convince someone, if he sees that the big "players" use it.

A list of possible/current big players would be:
- facebook (react, jest, create-react-app, immutable.js) \[[1]] \[[2]]
- babel \[[1]]
- netflix \[[3]]
- red badger \[[2]]

[1]: https://youtu.be/hkfBvpEfWdA?t=170
[2]: https://youtu.be/0Q4kUNx85_4?t=2223
[3]: https://github.com/prettier/prettier/pull/1104

**Comment by vjeux** [comment]
This would be awesome indeed :)

Some more: https://twitter.com/jlongster/status/852532847252566017

- large parts of healthcare.gov
- all zeit js projects
- haul soon react navigation
- growing % of mapbox
- playstation now
- glamorous
- gatsby
- starting to get momentum inside of Spotify
- thimble/brackets
- react native app of Khan Academy
- babili
- some projects at Uber
- pretty heavily at MLS (Major League Soccer)
- one internal project at Zoho
- all teams of new target.com platform

**Comment by codyzu** [comment]
If I put together a PR for this, would you want:

1. Just a list of the names? (maybe formatted into 2 columns with a table)
1. A list of logos for some of the bigger names on the list?

I guess the big question is do we need to ask permission to put a company's name on the list? Or if we can verify through npm dependencies, is that enough???

**Comment by codyzu** [comment]
👉 poke @vjeux

**Comment by vjeux** [comment]
As unfortunate as it is, I think having a list of ~10 recognizable names is going to be the most impactful. And yes, we need some sort of public acknowledgment that they are using it. Either a tweet by an employee or reaching out to them and asking if they'd be up to it.

**Comment by nickbalestra** [comment]
We are now using it @opentable for the [OpenComponents](https://github.com/opentable/oc) project -> https://github.com/opentable/oc/pull/507

**Comment by leipert** [comment]
My proposal would be names (or logos?) for the most commonly known (by my standards - no offense).

@vjeux probably can stand surety for:
- react 
- jest
- yarn

Then we have these OSS projects:
- babel (Is @hzoo d'accord with naming it on the prettier README?)
- zeit (@rauchg tweeted [evidence](https://twitter.com/rauchg/status/852675452812443648). Okay with naming zeit in README?)
- webpack (At the moment [webpack-cli](https://github.com/webpack/webpack-cli/blob/master/package.json#L47). Is @sokra fine with naming it in the README? )

Maybe we should leave out, as there are no OSS projects of my knowledge, that use it:
- khan acadamy
- netflix

AND a link to this issue, so that people can get an overview over everything else?

TL;DR I think react, jest, yarn, babel, zeit and webpack are big enough to stand alone on the prettier README ;)

**Comment by vjeux** [comment]
@leipert sorry, didn't see this until now. Go for it!
